Makefile.reconcilermanager

```diff
@@ -42,7 +42,7 @@ configsync-crds: "$(CONTROLLER_GEN)" "$(KUSTOMIZE)" "$(ADDLICENSE)"
 	&& rm ./manifests/configmanagement.gke.io_namespaceconfigs.yaml \
 	&& rm ./manifests/configmanagement.gke.io_repoes.yaml \
 	&& rm ./manifests/configmanagement.gke.io_syncs.yaml \
-	&& "$(ADDLICENSE)" ./manifests
+	&& "$(ADDLICENSE)" -y 2024 ./manifests
```

Comment:
I don't think this really addresses the issue as described - this will just set all CRDs to always have a 2024 license.

- If a new CRD is added in 2025 (for example), it should use 2025 as the year
- I assume we also want previously created CRDs to match the year they were introduced

Reply:
Agreed. The add-license code always picks the latest year, which is correct for new files. The problem is that they're only marked as new because we're deleting and re-generating them.

We might need a more advanced solution that "fixes" the copyright year after generation if the file already existed beforehand, maybe by using a `git diff --name-only --exit-code path/to/file` in a bash script or something.